bin/add_frontend_server.rb

```diff
@@ -156,21 +164,21 @@ def execute_ssh_on_channel(channel, command, exit_error_string)
                           ],
                       })
 
-puts "Created instance #{instance_id} with name #{instance_name}"
 
 private_dns_name = ec2client.describe_instances({instance_ids: [instance_id],}).reservations[0].instances[0].private_dns_name
-
-puts "Private DNS name #{private_dns_name}"
+puts "Created instance #{instance_id} with name #{instance_name} and private dns name #{private_dns_name}"
```

Comment:
On line 48 above, could you use the following to avoid the awkward \' escaping?

  puts ssh.exec!('echo "Verifying connection to gateway"')